styles/navigation/navbar/navbar.twig

```diff
@@ -0,0 +1,9 @@
+<ul class="navbar">
+  {%for item in items%}
+    <li class="navbar__item">
+      <a href="{{ item.url }}" class="navbar__link{% if item.active %} is-active{% endif %}">
```

Comment:
1. is-active on classi joka kuvastaa statea, eli kyllä
2. Tää on periaatteessa ihan validi huolenaihe, mutta en oo varma kuinka paljon selkeempi se olis että tän tilalla olis for looppi? Jos tää kuulostaa siltä että halutaan toteuttaa niin voin tehdä tän muutoksen.